src/nvim/api/vim.c

```diff
@@ -2241,7 +2241,7 @@ Dictionary nvim_eval_statusline(String str, Dict(eval_statusline) *opts, Error *
   if (highlights) {
     Array hl_values = ARRAY_DICT_INIT;
     const char *grpname;
-    char user_group[6];
+    char user_group[15];
```

Comment:
Why was it 6 before, and why is it 15 now?

Reply:
strlen(User) = 4
strlen(2147483647) = 10
10 + 4 + NIL

Reply:
Can you add that as a comment.

Reply:
I added a comment with that information.

Reply:
[780ea84](https://github.com/neovim/neovim/pull/23239/commits/780ea845f9022b85a33b86f35c19b7d29594a466)

Refers to the `UserX` hl groups, so 4 chars for `User`, 1 for the digit, and 1 for the NUL presumably.

Reply:
Naturally, GitHub didn't show me the new comments here :roll_eyes: